README.md

```diff
@@ -44,11 +44,11 @@ file into it. All changes should be made to this file. This folder is excluded i
 ### Database
 
 For the persistence needs of this application `h2` has been used.
-The database is configured to store files under `~/.snowdrop-bot/db`.
+In prod and default profiles, the database is configured to store files under `/data/snowdrop-bot/db/bot.db`.
```

Comment:
Can you also mention that with the `dev profile` a H2 in memory DB is then launched ?

Reply:
yes, done!

Reply:
I propose also to change the first line and to say `For our application and persistence needs, h2 has been used as Database.`